accepted/2020/windows-specific-apis/windows-specific-apis.md

```diff
@@ -133,8 +133,6 @@ Windows-specific:
 ### System.Diagnostics
 
 * **Process**
-    - `EnterDebugMode()`
-    - `LeaveDebugMode()`
```

Comment:
these two methods are `nop` on non-Windows, some time ago @stephentoub has suggested to not to mark `nop` are Windows-specific

Reply:
Makes sense.